mkosi/__init__.py

```diff
@@ -1843,6 +1843,16 @@ def extract_pe_section(context: Context, binary: Path, section: str, output: Pat
         )
 
 
+def want_signed_pcrs(config: Config) -> bool:
+    return bool(
+        config.sign_expected_pcr == ConfigFeature.enabled or
+        (
+            config.sign_expected_pcr == ConfigFeature.auto and
+            find_binary("systemd-measure", "/usr/lib/systemd/systemd-measure", root=config.tools())
+        )
+    )
```

Comment:
Wrapping this in `bool` does the trick, but that this is anything that a bool is just an artifact from or returning the first non-False thing and and returning the last. Checking the return from `find_binary` makes this intent a bit clearer, I think.
```suggestion
    return (
        config.sign_expected_pcr == ConfigFeature.enabled or
        (
            config.sign_expected_pcr == ConfigFeature.auto and
            find_binary("systemd-measure", "/usr/lib/systemd/systemd-measure", root=config.tools()) is not None
        )
    )
```